resources/schema.json

```diff
@@ -1,15 +1,40 @@
 {
     "$schema": "https://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "mutator_profile": {
+            "type": "boolean"
```

Comment:
The mutator profile can take the same properties as a mutator. e.g. the ignore properties.

Reply:
Yes I learned that later... I think I'm missing a rebase actually